opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -3253,6 +3253,26 @@ input OpinionAddInput {
   clientMutationId: String
   update: Boolean
 }
+input OpinionUserAddInput {
+  stix_id: String
+  x_opencti_stix_ids: [String]
+  opinion: String!
+  opinion_types: [String]
+  explanation: String
+  authors: [String]
+  revoked: Boolean
+  lang: String
+  confidence: Int
+  objectMarking: [String]
+  objectLabel: [String]
+  objectOrganization: [String]
+  externalReferences: [String]
+  objects: [String]
+  created: DateTime
+  modified: DateTime
+  clientMutationId: String
```

Comment:
Normally, this property is not used anymore

Reply:
Yes you're right: 
`clientMutationId` was a requirement for Relay Classic,  it is not required for Relay Modern.

Reply:
As there are many uses of `clientMutationId` on the platform, we have decided to keep it for now.